src/Alert/README.md

```diff
@@ -27,47 +27,50 @@ This component utilizes and extends the `Alert` component from react-bootstrap.<
 
 ```jsx live
 <>
-<Alert variant="success">
-  This is a "success" alert with{' '}
-  <Alert.Link href="#">an example link</Alert.Link>. Give it a click if you
-  like.
-</Alert>
-<Alert variant="info">
-  This is a "info" alert with{' '}
-  <Alert.Link href="#">an example link</Alert.Link>. Give it a click if you
-  like.
-</Alert>
-<Alert variant="danger">
-  This is a "danger" alert with{' '}
-  <Alert.Link href="#">an example link</Alert.Link>. Give it a click if you
-  like.
-</Alert>
-<Alert variant="warning">
-  This is a "warning" alert with{' '}
-  <Alert.Link href="#">an example link</Alert.Link>. Give it a click if you
-  like.
-</Alert>
+  <Alert variant="success">
+    This is a "success" alert with{' '}
+    <Alert.Link href="#">an example link</Alert.Link>. Give it a click if you
+    like.
+  </Alert>
+  <Alert variant="info">
+    This is a "info" alert with{' '}
+    <Alert.Link href="#">an example link</Alert.Link>. Give it a click if you
+    like.
+  </Alert>
+  <Alert variant="danger">
+    This is a "danger" alert with{' '}
+    <Alert.Link href="#">an example link</Alert.Link>. Give it a click if you
+    like.
+  </Alert>
+  <Alert variant="warning">
+    This is a "warning" alert with{' '}
+    <Alert.Link href="#">an example link</Alert.Link>. Give it a click if you
+    like.
+  </Alert>
 </>
 ```
 
 ### With Action Buttons
-The ``Alert`` component supports a dismissible button and a custom call-to-action button, via the ``dismissible`` and ``button`` props respectively. The buttons may be right aligned or stacked. On extra small screen widths, the buttons will be stacked.
+
+The `Alert` component supports a dismissible button and a custom call-to-action button, via the `dismissible` and `button` props respectively. The buttons may be right aligned or stacked. On extra small screen widths, the buttons will be stacked.
 
 The stacked variant should be used for:
+
 - Placement in sidebars or small container
 - Extra-small breakpoint and below
+
 ---
 
 ```jsx live
 <>
   <Alert
     variant="info"
-    actions={[
-      <Button>Hello</Button>,
-    ]}
+    actions={[<Button>Hello</Button>]}
```

Comment:
Can we not change this? The way it was before is more readable. Same in other places in this file